app/src/main/java/com/android/fundamentals/workshop03/LocationsAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.android.fundamentals.workshop03
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.android.fundamentals.R
+import com.android.fundamentals.domain.location.Location
+
+class LocationsAdapter : ListAdapter<Location, LocationViewHolder>(LocationDiffCallback()) {
```

Comment:
remove kt file in solution folder? and use this one there

Reply:
done